lib/Client.js

```diff
@@ -109,11 +109,12 @@ class Client extends EventEmitter {
     * @arg {Function} [options.reconnectDelay] A function which returns how long the bot should wait until reconnecting to Discord.
     * @arg {Number} [options.requestTimeout=15000] A number of milliseconds before requests are considered timed out. This option will stop affecting REST in a future release; that behavior is [DEPRECATED] and replaced by `options.rest.requestTimeout`
     * @arg {Object} [options.rest] Options for the REST request handler
-    * @arg {Object} [options.rest.agent] A HTTPS Agent used to proxy requests
+    * @arg {Object} [options.rest.agent] A HTTPS Agent (if https: true, default) or an HTTP agent (if https: false) used to proxy requests
```

Comment:
```suggestion
    * @arg {Object} [options.rest.agent] A HTTP used to proxy requests
```
this would probably be better said as `HTTP Agent`, noting that difference doesn't seem that important

Reply:
```suggestion
    * @arg {Object} [options.rest.agent] An HTTP/S Agent used to proxy requests
```